models/recharge__billing_history.sql

```diff
@@ -57,7 +57,7 @@ with orders as (
 ), joined_enriched as (
     select 
         joined.*,
-        total_price - total_refunds as total_net_order_value -- total_price includes taxes and discounts, so only need to subtract total_refunds to get net. 
+        total_price - total_refunds as total_net_order_value -- total_price includes taxes and discounts, so only need to subtract total_refunds to get net.  
```

Comment:
This calculation seems to be a bit off at points (particularly for transactions were they are prepaid). For example, these prepaid transactions typically show a negative total_net_order_value which I don't think is entirely accurate.

Additionally, there are a number of transactions where I see the total_net_order_value is just the cost of shipping. Should we be recording this? I imagine these are for subscriptions that were paid in full originally, but are now only being charged for shipping. What are your thoughts?